PRODUCTIONISATION.md

```diff
@@ -19,26 +19,26 @@
 
 Ensuring that your project adheres to the following RFCs is a good first step as these will be referenced further below:
 
+* [RFC 008 - Server naming convention][rfc_008]
 * [RFC 011 - Editorconfig][rfc_011]
 * [RFC 012 - Makefile standards][rfc_012]
 * [RFC 013 - Code repository standards][rfc_013]
 
 ### 1. General requirements
 
 * [ ] 1.1 Application source code hosted with a Git VCS service
-* 1.2 No linting violations
-  * [ ] 1.2.1 No style related linting violations and no overrides permitted
-  * [ ] 1.2.2 No complexity related linting violations. Overrides permitted but must be commented inline with the code it is overriding
-* [ ] 1.3 `.editorconfig` as per [RFC 011][rfc_011]
-* [ ] 1.4 `Makefile` as per [RFC 012][rfc_012]
-* [ ] 1.5 `README.md` as per [RFC 013][rfc_013]
-* [ ] 1.6 `README.md` to also include references for all 3rd party service integrations
-* 1.7 Local development seeds from production:
-  * [ ] 1.7.1 Seed a specific item (for example a product, in an ecom system)
-  * [ ] 1.7.2 Seed a specified number of items in batch
-  * [ ] 1.7.3 Seed full catalogue of items
-* [ ] 1.8 Local development seed fixtures (for example, using ffaker for Ruby/Rails)
-* [ ] 1.9 Install an application profiler for the `development` environment so that metrics such as page render times, database query times can be used to make performance improvements
+* [ ] 1.2 `.editorconfig` as per [RFC 011][rfc_011]
+* [ ] 1.3 `Makefile` as per [RFC 012][rfc_012]
+* [ ] 1.4 `README.md` as per [RFC 013][rfc_013]
+* [ ] 1.5 `README.md` to also include references for all 3rd party service integrations
+* 1.6 Local development seeds from production (or production-like data depending on project):
```

Comment:
Let's lose seeding from production, and instead only ask for "production like".

Very conscious of the fact that in most situations, we don't want production data on all of our machines, for security concerns ;)

Reply:
👍 - I think plan was for any production data to be obfuscated. But anything production like that can assist with debugging.

Reply:
Might also be worth mentioning that "production-like" is around having similar sized datasets, for diagnosing unusual performance regressions at scale, and not as a "getting started" thing, which is where the fakers noted below comes in

Reply:
Updated in https://github.com/madetech/productionisation/pull/51/commits/e363fecf86ec313b6e130896a70278698cad718a accordingly.